shynet/dashboard/templates/dashboard/includes/bar.html

```diff
@@ -0,0 +1,15 @@
+{% load helpers %}
+
+{% if use_relative_max %}
+    <div
+        class="absolute h-6"
+        style="width: {{count|percent:max}}; top: 6px; left: 0px; height: calc(100% - 12px); background-color: var(--color-urge-200-fallback)"
+    >
+    </div>
+{% else %}
+    <div
+        class="absolute h-6"
+        style="width: {{count|percent:total}}; top: 6px; left: 0px; height: calc(100% - 12px); background-color: var(--color-urge-200-fallback)"
+    >
+    </div>
+{% endif %}
```

Comment:
simplify to 
```
<div
    class="absolute h-6"
    style="width: {% if use_relative_max %}{{count|percent:max}}{% else %}{{count|percent:total}}{% endif %}; top: 6px; left: 0px; height: calc(100% - 12px); background-color: var(--color-urge-200-fallback)"
></div>
```
?

Reply:
I know it probably should be done this way. This line just looks so ugly.

Reply:
I know :p
It's ugly code vs duplicate code